src/frontend/js/components/PaymentScheduleGrid/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { DataList } from '@openfun/cunningham-react';
+import { StringHelper } from 'utils/StringHelper';
+
+export const PaymentScheduleGrid = () => {
+  return (
+    <div className="payment-schedule__grid">
+      <DataList
+        columns={[
+          {
+            id: 'date',
+            renderCell: (context) =>
+              context.row.id === 'total' ? <strong>{context.row.date}</strong> : context.row.date,
+          },
+          {
+            id: 'amount',
+            renderCell: (context) =>
+              context.row.id === 'total' ? (
+                <strong>{context.row.amount}</strong>
+              ) : (
+                context.row.amount
+              ),
```

Comment:
May be cleaner with css: 
```css
.total .amount, .total .date { font-weight: 600 }
```
I don't know if we've these kind of selectors.

Reply:
Right, this component is a bit zombie waiting for its real implementation for V2. +1